topostats/plottingfuncs.py

```diff
@@ -223,17 +236,20 @@ def save_figure(self):
                     format=self.save_format,
                     bbox_inches="tight",
                     pad_inches=0,
+                    dpi=self.dpi,
                 )
             else:
-                plt.savefig((self.output_dir / f"{self.filename}.{self.save_format}"), format=self.save_format)
+                plt.savefig(
+                    (self.output_dir / f"{self.filename}.{self.save_format}"), format=self.save_format, dpi=self.dpi
```

Comment:
I realise this isn't a change introduced with this PR but [savefig()](https://matplotlib.org/stable/api/_as_gen/matplotlib.pyplot.savefig.html) infers the `format` from the file-extension and as these are identical the code can be reduced to...

```suggestion
                    (self.output_dir / f"{self.filename}.{self.save_format}"), dpi=self.dpi
```